Dockerfile.agnos

```diff
@@ -26,7 +26,7 @@ RUN /tmp/agnos/base_setup.sh
 # ################## #
 # #### Compiler #### #
 # ################## #
-FROM agnos-base as agnos-compiler
+FROM agnos-base AS agnos-compiler
```

Comment:
These should not be part of this PR as they are unrelated to the uv introduction.

Reply:
@robin-reckmann please see previous comment https://github.com/commaai/agnos-builder/pull/253#discussion_r1698344560. The warning removal was requested and the FromAsCasing is also part of warnings.

Reply:
Ah yes, that comment was only for the newly introduced warning. But this part is already fixed on master anyway.